### ISSUE TITLE
Addition of vertical line for today's date

### ISSUE THREAD

**Issue by cmclinton** [giocomai/ganttrify#7]
In my local version of Ganttrify, I added the ability to mark today's date on the Gantt with a vertical line. Perhaps this could be added to the main code?

```
ganttrify <- function(project,
                      spots = NULL,
                      by_date = FALSE,
                      exact_date = FALSE, 
                      project_start_date = Sys.Date(),
                      colour_palette = wesanderson::wes_palette("Darjeeling1"),
                      font_family = "sans",
                      mark_quarters = FALSE,
                      size_wp = 6, 
                      size_activity = 4,
                      size_text_relative = 1,
                      month_number = TRUE,
                      colour_stripe = "lightgray",
                      todays_date = TRUE) {

```

```
# add a line for today's date
  if (todays_date == TRUE){
    gg_gantt <- gg_gantt +
      ggplot2::geom_vline(xintercept = Sys.Date(),color="grey50")

  }
```

**Comment by giocomai** [comment]
Yes, sounds meaningful, I used this myself to show current status in interim reports or internal documents. I'll definitely add something along these lines when I next update the package in the coming weeks. Thanks for the suggestion!

**Comment by paulditterline** [comment]
Love this idea!

**Comment by dmkaplan2000** [comment]
Hi, if I may, I would suggest a more flexible mechanism that allows putting vertical lines on top of the other Gantt data at arbitrary dates with labels. I have a case where I want the Gantt diagram for a proposal to show activity before the grant is actually awarded, so the Gantt chart will have a vertical line with a label "Project start" or something of the style.

**Comment by giocomai** [comment]
Yes, probably the implementation will be something like a `vertical_line` parameter that defaults to FALSE, that takes today's date if set to TRUE, but that will accept dates, or month numbers. 

Adding a nice label on top remains possible, but no so easy to generalise, which is partly why I didn't tackle this yet. 

I'll think this through, and probably add at least a basic implementation soon.

In the meantime, adding to the ouput: `+  ggplot2::geom_vline(xintercept = Sys.Date(),color="grey50")` remains a meaningful workaround.

**Comment by dmkaplan2000** [comment]
Yes, will use the workaround. As a quick fix, perhaps you could add a spot-style marker at the top of the line? Just a thought...
